### PR TITLE
Added better ASYNC support

### PR DESCRIPTION
**Pull request by jasonshortphd**
Changed tweet screen to use a constructor and then chain to an async initializer.  
Also changed the other async warning to an async delegate.

### CODE REVIEW

File: MyStocks.Forms/MyStocks.Forms.iOS/MyStocks.Forms.iOS.csproj [hunk 1]
```diff
@@ -171,6 +186,13 @@
       <Version>1.0.2</Version>
     </XamarinComponentReference>
   </ItemGroup>
+  <ItemGroup>
```
Comment (jamesmontemagno) [inline MyStocks.Forms/MyStocks.Forms.iOS/MyStocks.Forms.iOS.csproj]: I'm not sure what this adds... looks scary.

File: MyStocks.Forms/MyStocks.Forms.iOS/MyStocks.Forms.iOS.csproj [hunk 2]
```diff
@@ -16,6 +16,21 @@
     <RestorePackages>true</RestorePackages>
     <TargetFrameworkIdentifier>Xamarin.iOS</TargetFrameworkIdentifier>
     <TargetFrameworkVersion>v1.0</TargetFrameworkVersion>
```
Comment (jamesmontemagno) [inline MyStocks.Forms/MyStocks.Forms.iOS/MyStocks.Forms.iOS.csproj:18]: I think this all needs to be removed, not sure how this got in here.

Comment (jasonshortphd) [reply MyStocks.Forms/MyStocks.Forms.iOS/MyStocks.Forms.iOS.csproj:18]: I agree, not sure why that would be added - weird.  I was a simple code change, should have reviewed that.

Comment (jasonshortphd) [reply MyStocks.Forms/MyStocks.Forms.iOS/MyStocks.Forms.iOS.csproj:18]: I will remove the changes in the csproj - there should not need to be any.

**Comment by jasonshortphd** [comment]
Ok, lets try that again.  There is still a lock file there that is changed that I should probably exclude in the future as well.

The fix for async was pretty fun to change.  I tested on all three platforms and it works.

**Pull request merged**